.github/workflows/check_osx.yml

```diff
@@ -25,7 +25,7 @@ jobs:
 
       # - name: Check without default features
       #   run: cargo check --no-default-features --features notan/shaderc
-      #   if: matrix.os == 'macos-14'
+      #   if: matrix.os == 'macos-latest'
```

Comment:
using "latest" is dangerous, as runners can change and the workflow breaks. I would recommend compiling on the oldest available runner as you will exclude users that have older macos versions.